.github/workflows/ci.yml

```diff
@@ -220,6 +221,24 @@ jobs:
           make VERBOSE=1 -j${BUILD_CONCURRENCY} ${make_targets}
           ctest --timeout ${TEST_TIMEOUT} --output-on-failure ${ctest_flags}
 
+  dr-testing:
+    name: Run DR shp-tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Build docker image
+        run: docker build . -t dr-test -f .github/Dockerfile
+      - name: Run tests
+        run: |
+          docker run --name dr-test -d -v $(pwd):/repo dr-test sh -c "
```

Comment:
there is an option in docker "--rm" to remove container once "run" is finished, use it in CI in order to not leave rubbishes